django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.conf import settings
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+
+class Comment(models.Model):
+    parent_object = GenericForeignKey("parent_type", "parent_id")
```

Comment:
Did you have in mind we'd do comment replies via parenting another comment object? That's probably not a bad way to do it, just wondering if you took comment replies into account as well.

It might be worth setting up a [Generic Relation](https://docs.djangoproject.com/en/3.1/ref/contrib/contenttypes/#reverse-generic-relations) for getting the replies of this comment if that's the case.

Reply:
A challenge this approach might pose is that if we want to fetch _all_ comments related to a single package, including all replies, querying that efficiently will be difficult.

Reply:
I didn't really consider replies but I think there's more thought needed for that. Choosing between something like threads and inline replies would need to be done first as inline replies could really just be a quote block. If you did want explicit parent comments then I think having an optional `parent_comment` field would work well to improve querying and matches the structure of replies better in my opinion.

Reply:
Specific parent/child relation for comments is probably the way to go, however that could loove room for improvement on the `parent_object` naming. Maybe `commented_object`, `related_object`, `topic`, or `thread`?

Reply:
Should replies be part of this PR or added in the future?

Reply:
I'd include them here in the model already while we're at it. No need to render them yet though